Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/TeleportPointer.cs

```diff
@@ -384,14 +383,6 @@ public override void OnTeleportRequest(TeleportEventData eventData)
             }
         }
 
-        /// <inheritdoc />
-        public override void OnTeleportStarted(TeleportEventData eventData)
-        {
-            // Turn off all pointers while we teleport.
-            IsTeleportRequestActive = true;
```

Comment:
Seems like it might be good to keep this line, to prevent additional input while the teleport movement / animation is happening. Were you seeing it cause issues?

 https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/007187adfabe7c548055986c0f5bee1ba9a8e035/Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/TeleportPointer.cs#L254

Reply:
Nevermind! I see now you removed it because `BaseControllerPointer` has an identical block of code.